google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -345,6 +345,9 @@ public StatementResult statementAbortBatch() {
 
   @Override
   public StatementResult statementSetRPCPriority(RpcPriority rpcPriority) {
+    if (rpcPriority == RpcPriority.UNSPECIFIED) {
```

Comment:
(see also above)
I think it would be better to let this method accept a `com.google.spanner.v1.Priority` instead of `RpcPriority`, as that would remove the need to add `UNSPECIFIED` to the `RpcPriority` enum.